src/Avalonia.Base/Input/Raw/RawTouchEventArgs.cs

```diff
@@ -1,15 +1,26 @@
+using System;
+
 namespace Avalonia.Input.Raw
 {
     public class RawTouchEventArgs : RawPointerEventArgs
     {
         public RawTouchEventArgs(IInputDevice device, ulong timestamp, IInputRoot root,
             RawPointerEventType type, Point position, RawInputModifiers inputModifiers,
-            long touchPointId) 
+            long rawPointerId) 
             : base(device, timestamp, root, type, position, inputModifiers)
         {
-            TouchPointId = touchPointId;
+            RawPointerId = rawPointerId;
+        }
+
+        public RawTouchEventArgs(IInputDevice device, ulong timestamp, IInputRoot root,
+            RawPointerEventType type, RawPointerPoint point, RawInputModifiers inputModifiers,
+            long rawPointerId)
+            : base(device, timestamp, root, type, point, inputModifiers)
+        {
+            RawPointerId = rawPointerId;
         }
 
-        public long TouchPointId { get; set; }
+        [Obsolete("Use RawPointerId")]
+        public long TouchPointId { get => RawPointerId; set => RawPointerId = value; }
```

Comment:
RawPointerId should be existed in any pointer event arg. Not only in touch related.
So RawTouchEventArgs in overall is useless class now. But I didn't mark it obsolete.